core/com.b2international.snowowl.core.rest.tests/src/com/b2international/snowowl/core/rest/CodeSystemApiAssert.java

```diff
@@ -118,7 +118,8 @@ public static Json prepareCodeSystemCreateRequestBody(final String codeSystemId,
 			"toolingId", TOOLING_ID,
 			"oid", codeSystemId,
 			"language", "en",
-			"url", "link"
+			"url", "link",
+			"bundleId", IComponent.ROOT_ID
```

Comment:
Is this required here? The default should be `ROOT`.

Reply:
Updated the create request builder to fall back to the previously set bundle ID on `null` value.